src/vivarium_sodium_reduction/components/risks.py

```diff
@@ -83,15 +86,26 @@ def setup(self, builder: Builder):
     def sodium_effect_on_sbp_drop(self, index, sbp_drop_value):
         sodium_exposure = self.sodium_exposure(index)
         sodium_exposure_raw = self.sodium_exposure_raw(index)
+        sbp_exposure_raw = self.sbp_exposure_raw(index)
 
         # FIXME: this should go in the constants.py file
-        mmHg_per_g_sodium = (
-            5.8 / 6.0
-        )  # 5.8 (2.5, 9.2) mmHg decrease per 6g/day sodium decrease
+        # previously used 5.8 (2.5, 9.2) mmHg decrease per 6g/day sodium decrease for both (doi: 10.1002/14651858.CD004937.pub2 ?)
+        mmHg_per_g_sodium_for_low_sbp = (
+            1.0  # (.5, 1.49) mmHg from https://doi.org/10.1161/CIRCULATIONAHA.120.050371
```

Comment:
I would like to include the parameter uncertainty on this. @albrja or anyone, do you have an example of how we have done this before that I can adapt?  I used [a random seed hack in an old project](https://github.com/aflaxman/vivarium_dla/blob/main/vivarium_dla/components.py#L32-L34) that would work, but there must be something more standard by now.